File: library/src/main/java/com/custom/photoView/photoTap/IcsScroller.java
package com.custom.photoView.photoTap;

import android.annotation.TargetApi;
import android.content.Context;

import com.custom.photoView.photoTap.GingerScroller;

@TargetApi(14)
public class IcsScroller extends GingerScroller {

    public IcsScroller(Context context) {
        super(context);
    }

    @Override
    public boolean computeScrollOffset() {
        return mScroller.computeScrollOffset();
    }

}

File: library/src/main/java/com/custom/photoView/photoGesture/IcsScroller.kt
package com.custom.photoView.photoGesture

import android.annotation.TargetApi
import android.content.Context

import com.custom.photoView.photoGesture.GingerScroller

@TargetApi(14)
class IcsScroller(context: Context) : GingerScroller(context) {

    override fun computeScrollOffset(): Boolean {
        return mScroller.computeScrollOffset()
    }

}
